### PR TITLE
Bootstrap update (v5.3.3) for CSS and JS

### PR DESCRIPTION
**Pull request by zetxek**
- Update to Bootstrap 5.3.3 (from 4.3.1)
- Switch to Bootstrap Scss

### CODE REVIEW

**Comment by zetxek** [comment]
The prod build breaks the CSS: 

image

In the development version it looks ok 

image

**Comment by zetxek** [comment]
Taking shape:

https://github.com/user-attachments/assets/5bf79349-483e-4372-9889-ed7dd8ee6588

https://github.com/user-attachments/assets/90201ac1-7443-4617-87f3-3d746c38060f

The idea is to separate the bootstrap-specific files from the rest of the theme, so it can be updated easier (or split from bootstrap in the future).

Pending:
- applying PostCSS for CSS optimization
- review dark theme

**Comment by zetxek** [comment]
The page load seems to be comparable for the pre-release version and the current production one

- Left: https://adritian-demo-git-theme-update-update-theme-d564cd-adrianmoreno.vercel.app/ (preview URL for https://github.com/zetxek/adritian-demo/pull/93)
- Right: production version (https://adritian-demo.vercel.app/)

image

File: assets/scss/bootstrap/_variables.scss [hunk 1]
```diff
@@ -0,0 +1,1751 @@
...
+  "600": $gray-600,
+  "700": $gray-700,
+  "800": $gray-800,
+  "900": $gray-900
+) !default;
+// scss-docs-end gray-colors-map
+// fusv-enable
+
+// scss-docs-start color-variables
+$blue:    #478079 !default; // adritian override
```
Comment (zetxek) [inline assets/scss/bootstrap/_variables.scss:39]: Adritian custom blue

File: assets/scss/bootstrap/_variables.scss [hunk 2]
```diff
@@ -0,0 +1,1751 @@
...
+//
+// Define the maximum width of `.container` for different screen sizes.
+
+// scss-docs-start container-max-widths
+$container-max-widths: (
+  sm: 540px,
+  md: 720px,
+  lg: 960px,
+  xl: 1140px,
+  xxl: 1200px,
```
Comment (zetxek) [inline assets/scss/bootstrap/_variables.scss]: another override/customization on the defaults

**Comment by zetxek** [comment]
The demo site doesn't build right away due to some changes in the config - but seems to be doing well layout-wise:

https://github.com/zetxek/adritian-demo/pull/94

Updating to Bootstrap 5 with Sass will be worth it as the customizations are quite much cleaner, using `variables`.
Excited about this facelift to the theme!

**Comment by zetxek** [comment]
The speed is confirmed still in production 🎉 

Just tested adrianmoreno.info in mobile and desktop after https://github.com/zetxek/adrianmoreno.info/pull/235, and the score in lighthouse is still 100/100/100/100 (performance/accessibility/best practices/SEO)

**Comment by dcorto** [comment]
Nice @zetxek I will try to update my site during the week, amazing work 🔝

**Comment by zetxek** [comment]
> Nice @zetxek I will try to update my site during the week, amazing work 🔝

fantastic! If you find any issue - we can update the docs in https://github.com/zetxek/adritian-free-hugo-theme/blob/main/upgrading.md
For a while I didn't want to make any breaking changes - but I wonder how many people upgrade the theme... and even if they do, some improvements are worth the manual update!

**Comment by dcorto** [comment]
> > Nice @zetxek I will try to update my site during the week, amazing work 🔝
> 
> fantastic! If you find any issue - we can update the docs in https://github.com/zetxek/adritian-free-hugo-theme/blob/main/upgrading.md For a while I didn't want to make any breaking changes - but I wonder how many people upgrade the theme... and even if they do, some improvements are worth the manual update!

Hi @zetxek I updated my site with the last release, after following the upgrading.md all works great for now i didn't notice any break on my site. Thanks for that big update! :tada:

**Pull request merged**